Cartfile

```diff
@@ -0,0 +1,2 @@
+github "raphaelmor/Polyline" ~> 4.2
```

Comment:
This version is explicitly for swift 4. I'm not sure this will pass unless this changed to v4.1.1.

Reply:
I just upgraded the Bitrise apps to use Xcode 9 instead of Xcode 8.3. The builds should pass now.